includes/admin/tools/import/class-give-import-donations.php

```diff
@@ -232,9 +232,13 @@ class="button button-primary button-large button-secondary <?php echo esc_attr("
 		/**
 		 * Show success notice
 		 *
+         * @unreleased Add nonce check.
 		 * @since 1.8.14
 		 */
 		public function import_success() {
+            if( ! isset($_GET['_wpnonce']) || ! wp_verify_nonce( $_GET['_wpnonce'], 'give_donation_import_success' ) ) {
```

Comment:
This could probably be updated to use `check_ajax_referer()`  as well.

Reply:
At this point, this is not an ajax request. As the importer runs, the page URL changes for each step and this is the function that renders the last step.

That's the only reason I'm not using `check_ajax_request`.

Reply:
Oh, nice! Thanks for showing me that. Resolved on a66ce0680c798582735bc39067278fb3e1a847b2